pkg/storage/mysql/mysql_test.go

```diff
@@ -1,21 +1,113 @@
-package mysql_test
+package mysql
```

Comment:
Any reason for changing this?

Reply:
allow access to internal structure of mysql if needed (also be consistent with postgres_test.go as well).